src/components/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from 'next/link';
+import styles from './Header.module.css';
+import Image from 'next/image';
+import avatarSvg from '@/src/assets/avatar.svg';
+
+export default function Header() {
+  return (
+    <header className={styles.header}>
+      <div className={`${styles.headerContainer}`}>
```

Comment:
P3:
단일 클래스라면 아래처럼 하시는 걸 추천드려요~

```suggestion
      <div className={styles.headerContainer}>
```